### DISCUSSION TITLE
Is there a way to have utoipa not pick up the Rust docs

### DISCUSSION THREAD

**Discussion by dadepo** [juhaku/utoipa#664] [Q&A] [answered]
Hi, so I just introduced `utoipa` to a project and so far so good. 

The only thing is that I also have rust docs (written with ///) that documents the handler functions, and this by default get picked up by utoipa and added to the swagger documentation.

This is a undesirable since the rust docs also document the handler function interface (parameter and return values) and I would prefer not to have this show up in the swagger docs. I will want them to show up though when I generate the documentation by running cargo docs.

Is there a way to indicate to utoipa wihch part of the rust docs it should include or omit in the swagger ui?

**Comment by juhaku** [comment, marked as answer]
Hey, that is unfortunately currently not possible as it treats the doc comment `///` as a summary and a description. Here is a an issue related to this 439. There is a plan to have option to override these values in future. Currently only way is to manually split the description as demonstrated here https://github.com/juhaku/utoipa/issues/439#issuecomment-1504081544.
